packages/design-system-tokens/src/themes/core.ts

```diff
@@ -135,6 +135,10 @@ export const global: AnyTokenValues = {
   'site-margins-mobile':        '1.5rem',
   'site-max-width':             '1040px',
   'text-max-width':             '53rem',
+  'image-path':                 '"../images"',
+// add an error highlight to choice components inside a ChoiceList with an errorMessage.
+// TODO: remove this, we should not be setting variables like this in sass
+  'ds-include-choice-error-highlight': 'false',
```

Comment:
Could you explain a little more why this needed to be added to the theme variables? I don't understand

Reply:
I just needed a place to stick it since it didnt seem to fit anywhere.  Maybe a flag, maybe just an override in healthcare.. Open to suggestions.

Reply:
Maybe it would be simpler for it to stay [in the child Sass](https://github.com/CMSgov/design-system/blob/acc9e0f67940870462d0e0edd21022bae24921ea/packages/ds-healthcare-gov/src/styles/settings/_override.build.scss#L2) until we can implement that logic solely with CSS variables. As it stands now, we turn it into a token, which then gets converted into Sass only to be used in just Sass. Is the long-term solution to create a variable like `choice__border-color--error` and set it to the same thing as the normal border color in themes where we don't want to show an error highlight?

Reply:
Oh wait, we currently have that exact variable 😆 . So maybe we don't even need this variable if we update that `choice__border-color--error` in the relevant theme? And we could have a comment about why it's the same color as the normal one.

Reply:
Oh dang, perfecto